src/test/java/cucumber/steps/StepDef_jurorRecord.java

```diff
@@ -491,4 +491,44 @@ public void seeText_inSameRow_asText(String Weeks, String nextToText) {
             NAV.seeText_inSameRow_asText(dateFormat.format(newDate), nextToText);
         }
     }
+    @When("^I press view all expenses button$")
+    public void pressViewAllExpensesButton(){
+        JUROR_RECORD.pressViewAllExpensesButton();
```

Comment:
I reckon we need new stepdef and page objects for expenses. I know we're bad at it, but I think we should :)

Reply:
Have pulled expenses stuff from juror record and created a new stepdef and page object.